spec/index.bs

```diff
@@ -5219,17 +5260,18 @@ enum GPUStoreOp {
 
     : <dfn>drawIndexed(indexCount, instanceCount, firstIndex, baseVertex, firstInstance)</dfn>
     ::
+        Draws indexed primitives.
 
         <div algorithm="GPURenderEncoderBase.drawIndexed">
             **Called on:** {{GPURenderEncoderBase}} this.
 
             **Arguments:**
             <pre class=argumentdef for="GPURenderEncoderBase/drawIndexed(indexCount, instanceCount, firstIndex, baseVertex, firstInstance)">
-                indexCount:
-                instanceCount:
-                firstIndex:
-                baseVertex:
-                firstInstance:
+                indexCount: The number of indices to draw.
+                instanceCount: The number of instances to draw.
+                firstIndex: Index to begin drawing from.
```

Comment:
we should be careful here. We are in "indexed" drawing, which means we have indices of vertices. But this "firstIndex" is not *that* index. It's an index of an index.

Reply:
Oof, yeah. This feels awkward to phrase no matter what. Looking at other API equivalents:

 - **Vulkan**: `firstIndex` is the base index within the index buffer.
 - **D3D12**: `StartIndexLocation` The location of the first index read by the GPU from the index buffer.
 - **Metal**: `indexBufferOffset` Byte offset within indexBuffer to start reading indices from.

Metal's is the most precise, IMO, but it also doesn't map directly to WebGPU's attribute. Not sure how I feel about the other two, Vulkan's feels like it suffers from the same issue that you pointed out and D3D's "location" seems like it could be misunderstood to be a byte offset. I tried this:

"Offset in indices into the index buffer begin drawing from."

Still not thrilled with it, but I'm guessing I won't be perfectly satisfied with any phrasing. Thoughts? (Made a similar change to `firstVertex` in `draw()`).